docs/dev/openapi.yml

```diff
@@ -782,6 +782,9 @@ components:
     ServerStatus:
       type: object
       properties:
+        serverVersion:
+          type: string
+          description: Version of the server (commit or stable version number)
```

Comment:
```suggestion
          description: Version of the server (commit hash or stable version number)
```

Reply:
Why commit hash or stable version number? If some tool/client wants to process this data, it's rather bad if the commit hash is returned for some instances and the version number for other ones.

Reply:
After consulting the git manual, I think commit itself is the correct way. As the hash is just a way to identify the commit object.

Reply:
@ErikMichelson the main problem is: Not every commit has a version, but every version has a commit. From that perspective it makes sense to either split it, or fallback to the more correct commit. This is what we do right now already when to figure it if clients are still compatible after a reconnect.

Reply:
Maybe we should use `serverVersion` for the commit and add an attribute `serverVersionName` which contains the version string.